packages/@aws-cdk/aws-eks/README.md

```diff
@@ -1193,6 +1193,20 @@ cluster.addHelmChart('ExternalSecretsOperator', {
 });
 ```
 
+Helm chart can come with Custom Resource Definitions (CRDs) defined that by default will be installed by helm as well. However in special cases it might be needed to skip the installation of CRDs, for that the property `skipCrds` can be used.
+
+```ts
+declare const cluster: eks.Cluster;
+// option 1: use a construct
+new eks.HelmChart(this, 'NginxIngress', {
+  cluster,
+  chart: 'nginx-ingress',
+  repository: 'https://helm.nginx.com/stable',
+  namespace: 'kube-system',
+  skipCrds: true,
```

Comment:
Is CRDs a typical convention for this? If not, I'd prefer to have this spelled out so it's more clear.

Reply:
Good point, I checked the docs of k8s and helm and it seems that this is a common convention.

[k8s custom resource definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
[helm custom resource definition](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/)

Reply:
In that case, I consider this good to go!